### ISSUE TITLE
Class names are not passed to webviews

### ISSUE THREAD

**Issue by victorporof** [mozilla/tofino#147]
...so all code that relies on `webview.webview-{index}` selectors (e.g. tab ordering, drag & drop etc.) doesn't work.

**Comment by victorporof** [comment]
Turns out drag'n'drop doesn't work for other reasons (it's a WIP), but the class names still aren't properly passed in, so we should fix that.
